### DISCUSSION TITLE
I can't sign in with my email

### DISCUSSION THREAD

**Discussion by ShezmuTeam** [vercel/community#3882] [Account Issue] [answered]
### Question

ShezmuTeam@proton.me

Here's my vercel account email
I am going to login and connect my GitHub account to deploy my app
But unfortunately, not able to login with this email
Would you plz fix it?
Sincerely

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble logging in. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Comment by itzmiftah** [comment]
Here's my vercel account email
I am going to login and connect my GitHub account to deploy my app
But unfortunately, not able to login with this email
Would you plz fix it?
Sincerely
